src/extra/python/scripts/calendar_calc.py

```diff
@@ -1,4 +1,4 @@
-from netcdftime import utime
+from cftime import utime
```

Comment:
I've also made a small update to `calendar_calc` to reflect the change of the `netcdftime` module to be `cftime`.